From 84a4677797d5c6cfae949f963aafd2fb3260d67a Mon Sep 17 00:00:00 2001
From: Szabolcs Nagy <nsz@port70.net>
Date: Tue, 29 Aug 2017 19:46:46 +0200
Subject: [PATCH] s390x: add bits/hwcap.h

based on linux arch/s390/include/asm/elf.h
(these macros should be exported into uapi, but they are not)
---
 arch/s390x/bits/hwcap.h | 15 +++++++++++++++
 1 file changed, 15 insertions(+)
 create mode 100644 arch/s390x/bits/hwcap.h

diff --git a/arch/s390x/bits/hwcap.h b/arch/s390x/bits/hwcap.h
new file mode 100644
index 00000000..ecc6ce1f
--- /dev/null
+++ b/arch/s390x/bits/hwcap.h
@@ -0,0 +1,15 @@
+#define HWCAP_S390_ESAN3	1
+#define HWCAP_S390_ZARCH	2
+#define HWCAP_S390_STFLE	4
+#define HWCAP_S390_MSA		8
+#define HWCAP_S390_LDISP	16
+#define HWCAP_S390_EIMM		32
+#define HWCAP_S390_DFP		64
+#define HWCAP_S390_HPAGE	128
+#define HWCAP_S390_ETF3EH	256
+#define HWCAP_S390_HIGH_GPRS	512
+#define HWCAP_S390_TE		1024
+#define HWCAP_S390_VXRS		2048
+#define HWCAP_S390_VXRS_BCD	4096
+#define HWCAP_S390_VXRS_EXT	8192
+#define HWCAP_S390_GS		16384
-- 
2.14.1

